src/redux/actions/dataActions.js

```diff
@@ -95,6 +115,18 @@ export const dataBlockUpdate = (blockData, blockId) => (dispatch) => {
 };
 
 // delete
```

Comment:
first argument of dataBlockDelete is an empty {}, should remove this arg

Reply:
Yes